gender_novels/analysis/gender_pronoun_freq_analysis.py

```diff
@@ -372,6 +386,156 @@ def sort_every_year(frequency_dict):
 
     return every_year_dict
 
+
+def box_gender_pronoun_freq(freq_dict, my_pal, title, x="N/A"):
+    """
+    Takes in a frequency dictionaries and exports its values as a bar-and-whisker graph
+    :param freq_dict: dictionary of frequencies grouped up
+    :param my_pal: palette to be used
+    :param title: title of exported graph
+    :param x: name of x-vars
+    :return:
+    """
+
+    plt.clf()
+    groups = []
+    val = []
+    for k, v in freq_dict.items():
+        temp = [k]*len(v)
+        groups.extend(temp)
+        val.extend(v)
+
+    df = pd.DataFrame({x: groups, 'Frequency': val})
+    df = df[[x, 'Frequency']]
+    sns.boxplot(x=df[x], y=df['Frequency'],
+                palette=my_pal).set_title("Relative Frequency of Female Pronouns to Total Pronouns")
+    plt.xticks(rotation=90)
+    # plt.show()
+
+    filepng = "visualizations/" + title + ".png"
+    filepdf = "visualizations/" + title + ".pdf"
+    plt.savefig(filepng, bbox_inches='tight')
+    plt.savefig(filepdf, bbox_inches='tight')
+
+    return
```

Comment:
Hi, this isn't pertinent, but if you aren't returning anything, this statement is unnecessary